microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -32,53 +32,113 @@ See:
 
 The Payment Request API⁠⁠ handles the actual payment transaction when a purchase is made by a user.  The Payment Request API uses the item details that the Digital Goods API provides, to make the in-app purchase by using whichever billing payment method the user has set up at the Microsoft Store.
 
-See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
+See:
+* [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
+
+
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, do any of the following:
```

Comment:
if the PWA Dev registers for the origin trial and uses the OT token at their site, can end-users use Edge Stable now to use the Digital Goods API features?
@runyuany